app/main.py

```diff
@@ -1,8 +1,46 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int, brand: str) \
```

Comment:
Please fix as per styling guidelines - each param must be on new line, also we don't use `\` as a line breaker